css/styles.css

```diff
@@ -0,0 +1,201 @@
+* {
+  font-family: Merriweather, Source Sans Pro, Oswald;
+  font-weight: lighter;
+}
+
+#container {
+  width: 875px;
+  margin: auto;
+ }
+
+header {
+  background-color: black;
+  width: 100%;
+  height: 100px;
+  border-bottom: 3px solid yellow;
+  display: flex;
+  padding-bottom: 10px;
+  float: left;
+ }
+
+ nav{
+  text-align: right;
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-around;
+ } 
+
+ .logo{ 
+  color: yellow;
+  font-size: 28px;
+  padding: 15px;
+  }
+
+ .tagline {
+  background-color: lightgray;
+  width: 100%;
+  height: 10%;
+  justify-content: center;
+  float: left;
+  margin-top: 0px;
+ }
+
+ .tagline p {
+  text-align: center;
+  font-style: italic;
+  color: black;
+  vertical-align: middle;
+  margin-bottom: 10px;
+ }
+
+span-taste{
+  color: grey;
+  text-decoration: line-through;
+}
+span-are{
+  text-decoration: underline yellow;
+}
+
+ a {
+  color: white;
+  text-decoration: none;
+  font-family: impact, sans-serif;
+  padding-right: 20px;
+ }
+
+ .standout-font{
+  text-transform: uppercase;
+  font-size: 18px;
+  font-weight: bolder;
+ }
+
+ a:hover {
+  color: yellow;
+ }
+
+footer {
+  background-color: black;
+  border: 1px black solid;
+  display: flex;
+  height: 40px;
+  float: left;
+  width: 100%;
+ /* font-family: sans-serif;*/
+ }
+
+ h4 {
+  vertical-align: middle;
+  font-family: sans-serif;
+  padding-left: 10px;
+  color: white;
+ }
+
+.main {
+  background-image: url("../images/coworking.jpg");
+  background-position: center;
+  height: 83vh;
+  width: 100%;
+  float: left;
+}
+
+.main-overlay {
+  background-color: grey 
+  opacity: 0.7;
+  color: black;
+  border: 1px solid lightgray;
+  width: 250px;
+  height: 400px;
```

Comment:
@AmyMirisola remove the height. let the content dictate the height of the component and use margin and padding to give spacing and move other elements 🔨